Samples/Hast.Samples.Consumer/Hast.Samples.Consumer.csproj

```diff
@@ -45,6 +45,9 @@
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe">
       <Version>4.7.0</Version>
     </PackageReference>
+    <PackageReference Include="Terminal.Gui">

```

Comment:
Nem érdemes ide hozzáadni?

Reply:
Nem értem a kérdést.

Reply:
Mert nem olvastad el figyelmesen!

Reply:
A link nem volt ott, úgy nehéz. 😖

Reply:
Nem vagyok benne biztos, kicsit felemás tapasztalataim vannak vele. Amikor minden klappol akkor remek. Viszont bizonyos funkciók (pl dialog box és fill layout) nem tudtam működésre bírni és nem tudom még eldönteni, hogy a hiba velem van vagy a libraryval.

Reply:
Én is ezt mondanám, ha nem olvastam volna el figyelmesen :D.

Ezekre amúgy rákérdeztél ott GitHubon? Discussions is van.

Reply:
Jó lesz ez, csak ki kellett tapasztalni. Ha nem azt csinálod ami a demójukban van akkor működik. 😂